pkg/api/server/register_containers.go

```diff
@@ -21,6 +21,19 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	//      name: name
 	//      type: string
 	//      description: container name
+	//    - in: body
+	//      name: body
+	//      description: Container to create
+	//      schema:
+	//        allOf:
+	//          - $ref: "#/definitions/ContainerConfig"
+	//          - type: object
+	//            properties:
+	//              HostConfig:
+	//                $ref: "#/definitions/HostConfig"
+	//              NetworkingConfig:
+	//                $ref: "#/definitions/NetworkingConfig"
+	//      required: true
```

Comment:
Without this parameter, you can't provide a configuration for the container you want to create.

Reply:
agreed